bazaarvoice/emodb
# PD-195923 fix jersey2 client dependencies

## Github Issue #

-

## What Are We Doing Here?

- set correct dependency versions for jersey2 maven modules
- adjust jersey2 class names
- adjust jersey2 package names
- adjust jersey2 module names

## How to Test and Verify

1. Check out this PR
2. Run `mvn -e -Ddependency-check.skip=true verify`
3. Verify that all tests pass
4. Profit

## Risk

### Level 

`Low`

### Required Testing

`Manual`

### Risk Summary
-

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
